x-pack/metricbeat/module/prometheus/remote_write/data.go

```diff
@@ -192,19 +190,6 @@ func (g remoteWriteTypedGenerator) GenerateEvents(metrics model.Samples) map[str
 	return eventList
 }
 
-// rateCounterUint64 fills a counter value and optionally adds the rate if rate_counters is enabled
```

Comment:
Why is this deleted?

Reply:
It's not used at all and linter complains about this :)